src/main/java/com/icthh/xm/tmf/ms/resource/config/DefaultProfileUtil.java

```diff
@@ -1,11 +1,12 @@
 package com.icthh.xm.tmf.ms.resource.config;
 
 import io.github.jhipster.config.JHipsterConstants;
-import java.util.HashMap;
-import java.util.Map;
 import org.springframework.boot.SpringApplication;
 import org.springframework.core.env.Environment;
 
+import java.util.HashMap;
```

Comment:
Unnecessary changes, according checkstyle imports orders must be the same as in old version